packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -65,7 +65,10 @@ export interface IOdspResponse<T> {
 	duration: number;
 }
 
-export interface TokenFetchOptionsEx extends TokenFetchOptions {
+export interface TokenFetchOptionsEx {
```

Comment:
What is the reason for removing extends TokenFetchOptions?

Reply:
The way this interface is being used is to communicate refresh, claims, tenantId and error for a getWithRetryForTokenRefresh utility. 
It didn't make any sense to force that utility to know about "request" etc.  All that utility needs to know/do is pass the stuff related to retrying around.